### ISSUE TITLE
bug: on ion-select can not set aria-label on ion-select-option

### ISSUE THREAD

**Issue by mastergogo** [ionic-team/ionic-framework#22917]
Ionic version:
5.x

Current behavior:
ion-select when used along with ion-select-option and when the interface is action-sheet then if we give aria-label to ion-select-options, for some reason browser changes the markup and does not allow the aria-label to be there on ion-select-options.

Expected behavior:
We should be able to provide the aria-labels to ion-select-options when used with an action-sheet as an interface.

Please find the working repo [here](https://github.com/mastergogo/ion-repo-ion-select): on-page tab3.vue there is an example

Let me know if more info is needed.

**Comment by liamdebeasi** [comment]
Thanks for the issue. The problem here is that `ion-select-option` is not actually rendered in your action sheet when opening `ion-select`, so anything you provide on `ion-select-option` will not be passed through other than the defined props.

**Comment by mastergogo** [comment]
Thanks @liamdebeasi  for responding, is there any way to give aria-label to ion-select-options?

**Comment by liamdebeasi** [comment]
There is no workaround at the moment.
